src/js/actions/layers.js

```diff
@@ -821,10 +821,13 @@ define(function (require, exports) {
      * @param {string=} modifier Way of modifying the selection. Possible values
      *  are defined in `adapter/src/lib/layer.js` under `select.vals`.
      *  Default is similar to "select", but the pivot layer will always be cleared.
+     * @param {boolean?} quiet If set, will prevent dispatches as result of this select
```

Comment:
Confusingly, nullable types are written `?T` not `T?` whereas optional types are written `T=`. I think you actually want `boolean=` here.